### ISSUE TITLE
How can I use it with tensorflow?

### ISSUE THREAD

**Issue by apollo-time** [deepinsight/insightface#21]
I see you got very good accuracy on LFW, and it is better than [@davidsandberg's](https://github.com/davidsandberg/facenet).
How can I use your pretraind model with tensorflow as like @davidsandberg's?
Thanks.

**Comment by nttstar** [comment]
Not supported currently.  We will be happy if someone can contribute a tensorflow/caffe version, like insightface-tensorflow,  insightface-caffe,  ^^.
